app/src/androidTest/java/fr/free/nrw/commons/AboutActivityTest.kt

```diff
@@ -104,6 +104,15 @@ class AboutActivityTest {
                 IntentMatchers.hasData(Urls.CREDITS_URL)))
     }
 
+    @Test
+    @Ignore("Fix Failing Test")
```

Comment:
@Rishavgupta12345 I just noticed that you are not actually running this test. Please remove the `@Ignore` annotation and fix the test.

Reply:
Most of the test in this file has the @Ignore annotation, I'm not sure if it is intended.

Reply:
@4D17Y4  https://github.com/commons-app/apps-android-commons/pull/4832 should fix most of the tests

Reply:
I have also fixed the recent test added by @Rishavgupta12345